### DISCUSSION TITLE
24v power supply

### DISCUSSION THREAD

**Discussion by baddivot** [nliaudat/esp32_8ch_motor_shield#69] [Ideas]
Hi nilaudat,
great work, thanks for designing and debugging! Would it be possible to equip the board with a 24V supply? I have 4 heating circuit distributors with a Homematic IP wired bus cable installed, which delivers 24V.

Thanks & Regards

**Comment by nliaudat** [comment]
Hi, 
I mean the best solution is to use a 24v To 5v Dc Converter
